.gitignore

```diff
@@ -11,6 +11,13 @@
 /*tags
 /autom4te.cache
 
+# generated files
```

Comment:
This shouldn't be part of this PR.

Reply:
You want a separate PR for it?

Reply:
I'd just avoid changing this at all, you can always put this in your `.git/info/exclude`.

And, FWIW, I strongly advise to always build outside of the source directory, then you won't have these files in the repository at all.